### DISCUSSION TITLE
Nested Collections

### DISCUSSION THREAD

**Discussion by rocco-haro** [excalidraw/excalidraw#5284] [Polls]
I absolutely love excalidraw. I tell everyone about it. 

Recently, I upgraded to excalidraw plus because I was just drawing EVERYTHING into a single scene. It's a beautiful monster. I also had a bit of a scare of losing my work... yay cloud!

Anyways, I'm a really heavy user and organizing my sh*t is something I need in order to keep myself sane. As a solo founder there are so many damn moving pieces, especially considering that we are a deep tech startup. 

I'm making this post as a feature request to allow Excaldraw+ users the ability to Nest Collections. Or create folders, and then put collections inside the folders, or whatever you want to call them. 

here's a screen shot of a current collection that I'll be breaking out into multiple collections soon... but the problem ill have soon is having a HUGE list of collections... would be great if I could organize the collections and give the "parent" collection a name :) 
![Screen Shot 2022-06-06 at 3 02 16 PM](https://user-images.githubusercontent.com/17133963/172257115-ad2ff768-3480-4311-9a81-173354dd5b84.png)

Keep up the awesome work & thank you for the consideration!

**Comment by louis030195** [comment]
I'm a big user of Obsidian + Notion + Excallidraw (all paid plans) and I have a large amount of drawing now, that does not scale with a 1d structure, so I'd love folders or some other kind of structure too ❤️

**Comment by rdasan-alaska** [comment]
My company just Got us Excalidraw subscriptions for All Our various Teams. We have a Collection Per Team. But I would like to have Sub folders within Each Collection For each separate Project that My Team does. Having SubFolders will help us organize better

**Comment by maielo** [comment]
Yes this is planned :), we will do it as first thing, once we finish migration to new database.

**Reply by rdasan-alaska** [reply]
@maielo  Any update on this?

**Reply by shay-berman** [reply]
Any updates?  we also need this subfolders

**Reply by Mrazator** [reply]
It remains as our top priority, but we don't have ETA yet. It might seem simple, but the request is coupled with pre-requisite to switch to a different database provider (from document to relational database), so it might take some time.

**Reply by skipper09** [reply]
Any update on this?
